### DISCUSSION TITLE
关于字体

### DISCUSSION THREAD

**Discussion by ssunxiangrui** [imsyy/home#248] [General]
我知道需要把原来的字体替换掉，并且这么做了
但是我用的是谷歌浏览器，ctlr+F5的字体是正常的
再次进去就变了样（坏掉了），比如 "u" 就没有相应的字体
（我表述的可能不太清楚）
***
规范一下：
前提条件：我把front已经改掉了
现在的情况：
使用谷歌浏览器直接进去，字体没有变化，坏了的
ctrl+F5清除缓存刷新就好了
但是F5普通刷新就又坏了
换一个浏览器就好了
请问：是缓存的问题吗？但是我已经清除过缓存了

**Comment by imsyy** [comment]
PWA 问题，等待缓存载入完成就行了
